packages/now-cli/src/commands/env/index.ts

```diff
@@ -0,0 +1,108 @@
+
+import add from './add';
+import pull from './pull';
+import ls from './ls';
+import rm from './rm';
+
+const help = () => {
+  console.log(`
+  ${chalk.bold(`${logo} now env`)} [options] <command>
+
+  ${chalk.dim('Commands:')}
+
+    add     [key] [environment]        Add environment variable (see below for examples)
+    pull    [filename]                 Read development environment from the cloud and write to a file, default .env
+    rm      [key] [environment]        Remove environment variable
+    ls      [environment]              List all variables for the specified environment
+
+  ${chalk.dim('Options:')}
+
+    -h, --help                     Output usage information
+    -A ${chalk.bold.underline('FILE')}, --local-config=${chalk.bold.underline(
+    'FILE'
+  )}   Path to the local ${'`now.json`'} file
+    -Q ${chalk.bold.underline('DIR')}, --global-config=${chalk.bold.underline(
+    'DIR'
+  )}    Path to the global ${'`.now`'} directory
+    -d, --debug                    Debug mode [off]
+    -t ${chalk.bold.underline('TOKEN')}, --token=${chalk.bold.underline(
+    'TOKEN'
+  )}        Login token
+
+  ${chalk.dim('Examples:')}
+
+  ${chalk.gray('–')} Add a new variable to multiple environments
+
+      ${chalk.cyan('$ now env add <name>')}
+      ${chalk.cyan('$ now env add API_TOKEN')}
+
+  ${chalk.gray('–')} Add a new variable for a specific environment
+
+      ${chalk.cyan('$ now env add <name> <production | preview | development>')}
+      ${chalk.cyan('$ now env add DB_CONNECTION production')}
+
+  ${chalk.gray('–')} Add a new environment variable from stdin
+
+      ${chalk.cyan(
+        '$ cat <file> | now env add <name> <production | preview | development>'
+      )}
+      ${chalk.cyan('$ cat ~/.npmrc | now env add NPM_RC preview')}
+
+  ${chalk.gray('–')} Remove a variable from multiple environments
+
+      ${chalk.cyan('$ now env rm <name>')}
+      ${chalk.cyan('$ now env rm API_TOKEN')}
+
+  ${chalk.gray('–')} Remove a variable from a specific environment
+
+      ${chalk.cyan('$ now env rm <name> <production | preview | development>')}
+      ${chalk.cyan('$ now env rm NPM_RC preview')}
+`);
+};
+
+const COMMAND_CONFIG = {
+  add: ['add'],
+  pull: ['pull'],
+  ls: ['ls', 'list'],
+  rm: ['rm', 'remove'],
+};
+
+export default async function main(ctx: NowContext) {
+  let argv;
+
+  try {
+    argv = getArgs(ctx.argv.slice(2), {});
+  } catch (error) {
+    handleError(error);
+    return 1;
+  }
+
+  if (argv['--help']) {
+    help();
+    return 2;
+  }
+
+  const output = createOutput({ debug: argv['--debug'] });
+  const { subcommand, args } = getSubcommand(argv._.slice(1), COMMAND_CONFIG);
+  switch (subcommand) {
+    case 'add':
+      return add(ctx, argv, args, output);
+    case 'pull':
+      return pull(ctx, argv, args, output);
+    case 'rm':
+      return rm(ctx, argv, args, output);
+    case 'ls':
+      return ls(ctx, argv, args, output);
+    default:
+      help();
```

Comment:
Should we write an error message here (eg. `now env set` does not exist`)?

Reply:
This help message is shown when you do `now env`, not `now env set`.

Are you saying you want to add an error message in addition to the help message or replace the help message with an error message?

Reply:
I was thinking of something on top of the help message, like that:
![image](https://user-images.githubusercontent.com/6616955/78400693-7fd10c80-75f7-11ea-9754-51779f8ad07d.png)

Reply:
Great idea, I added the error message and created a helper function so we can use in other commands.